docs/src/appendix/sncast/script/run.md

```diff
@@ -0,0 +1,14 @@
+# `run`
+Compile and run a cairo deployment script.
+
+## `<SCRIPT_NAME>`
```

Comment:
```suggestion
## `<MODULE_NAME>`
```

I'd change it back tbh, because it's a bit vague what script name is - I expect people to know what a module is though